src/main/java/sirius/biz/storage/layer1/replication/jdbc/SQLReplicationTaskStorage.java

```diff
@@ -42,6 +42,8 @@
      */
     public static final String FRAMEWORK_JDBC_REPLICATION = "biz.storage-replication-jdbc";
 
+    private static final String TXN_ID = "txnId";
```

Comment:
Optional: Rename `TXN_ID` to `TRANSACTION_ID`. I am not really a fan of the shortened field name but at least name the constant more descriptive please